### PR TITLE
IAM Updates

### PR DESCRIPTION
**Pull request by dantavori**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
- Changed the display names of `x_user_enabled` parameters (`x` is a CRUD command).
- Added the **Allow enabling users** parameter.
- Added support for customizing fields mapping differently in ***iam-create-user*** and ***iam-update-user*** commands.
   - Updated the outgoing mappers to use the ***User Profile - Create*** and ***User Profile - Update*** incident types. 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by OrelHaim — APPROVED** [review]
Looks amazing! 👍

File: Packs/Active_Directory_Query/ReleaseNotes/1_2_0.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+##### Active Directory Query v2
+- Added support for customizing fields mapping differently in ***iam-create-user*** and ***iam-update-user*** commands.
```
Comment (roysagi) [inline Packs/Active_Directory_Query/ReleaseNotes/1_2_0.md:4]:
```suggestion
- Added support for customizing fields mapping differently in the ***iam-create-user*** and the ***iam-update-user*** commands.
```

File: Packs/Active_Directory_Query/ReleaseNotes/1_2_0.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+#### Mappers
+##### User Profile - Active Directory (Outgoing)
+- Updated the mapper to use the ***User Profile - Create*** and ***User Profile - Update*** incident types.
```
Comment (roysagi) [inline Packs/Active_Directory_Query/ReleaseNotes/1_2_0.md:8]:
```suggestion
- Updated the mapper to use the **User Profile - Create** and **User Profile - Update** incident types. 
```

File: Packs/Attlasian/ReleaseNotes/1_1_0.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Atlassian IAM
+- Added the **Allow enabling users** parameter.
```
Comment (roysagi) [inline Packs/Attlasian/ReleaseNotes/1_1_0.md:4]:
```suggestion
- Added the *Allow enabling users* integration parameter.
```
you might want to mention the param name changes as well.

File: Packs/Attlasian/ReleaseNotes/1_1_0.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+- Added support for customizing fields mapping differently in ***iam-create-user*** and ***iam-update-user*** commands.
```
Comment (roysagi) [inline Packs/Attlasian/ReleaseNotes/1_1_0.md:5]:
```suggestion
- Added support for customizing fields mapping differently in the ***iam-create-user*** and the ***iam-update-user*** commands.
```

File: Packs/GitHub/ReleaseNotes/1_3_0.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### GitHub IAM
+- Added the **Allow enabling users** parameter.
```
Comment (roysagi) [inline Packs/GitHub/ReleaseNotes/1_3_0.md:4]: not seeing this param in your change.
but you do change other integration params - might want to mention that.

File: Packs/GitHub/ReleaseNotes/1_3_0.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+- Added support for customizing fields mapping differently in ***iam-create-user*** and ***iam-update-user*** commands.
```
Comment (roysagi) [inline Packs/GitHub/ReleaseNotes/1_3_0.md:5]:
```suggestion
- Added support for customizing fields mapping differently in the ***iam-create-user*** and the ***iam-update-user*** commands.
```

File: Packs/HelloIAMWorld/ReleaseNotes/1_1_0.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+##### Hello World IAM
+- Added the **Allow enabling users** parameter.
```
Comment (roysagi) [inline Packs/HelloIAMWorld/ReleaseNotes/1_1_0.md:4]:
```suggestion
- Added the *Allow enabling users* integration parameter.
```

File: Packs/HelloIAMWorld/ReleaseNotes/1_1_0.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+- Added support for customizing fields mapping differently in ***iam-create-user*** and ***iam-update-user*** commands.
```
Comment (roysagi) [inline Packs/HelloIAMWorld/ReleaseNotes/1_1_0.md:5]:
```suggestion
- Added support for customizing fields mapping differently in the ***iam-create-user*** and the ***iam-update-user*** commands.
```

File: Packs/HelloIAMWorld/ReleaseNotes/1_1_0.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
...
+
+#### Mappers
+##### User Profile - HelloIAMWorld (Outgoing)
+- Updated the mapper to use the ***User Profile - Create*** and ***User Profile - Update*** incident types.
```
Comment (roysagi) [inline Packs/HelloIAMWorld/ReleaseNotes/1_1_0.md:9]:
```suggestion
- Updated the mapper to use the **User Profile - Create** and **User Profile - Update** incident types. 
```

File: Packs/IAM-SCIM/ReleaseNotes/1_1_0.md [hunk 10]
```diff
@@ -0,0 +1,4 @@
+
+#### Mappers
+##### User Profile - SCIM (Outgoing)
+- Updated the mapper to use the ***User Profile - Create*** and ***User Profile - Update*** incident types.
```
Comment (roysagi) [inline Packs/IAM-SCIM/ReleaseNotes/1_1_0.md:4]:
```suggestion
- Updated the mapper to use the **User Profile - Create** and **User Profile - Update** incident types. 
```

File: Packs/Okta/ReleaseNotes/2_2_0.md [hunk 11]
```diff
@@ -0,0 +1,8 @@
+#### Integrations
+##### Okta IAM
+- Added the **Allow enabling users** parameter.
```
Comment (roysagi) [inline Packs/Okta/ReleaseNotes/2_2_0.md:3]:
```suggestion
- Added the *Allow enabling users* integration parameter.
```

File: Packs/Okta/ReleaseNotes/2_2_0.md [hunk 12]
```diff
@@ -0,0 +1,8 @@
...
+- Added support for customizing fields mapping differently in ***iam-create-user*** and ***iam-update-user*** commands.
```
Comment (roysagi) [inline Packs/Okta/ReleaseNotes/2_2_0.md:4]:
```suggestion
- Added support for customizing fields mapping differently in the ***iam-create-user*** and the ***iam-update-user*** commands.
```

File: Packs/Okta/ReleaseNotes/2_2_0.md [hunk 13]
```diff
@@ -0,0 +1,8 @@
...
+
+#### Mappers
+##### User Profile - Okta (Outgoing)
+- Updated the mapper to use the ***User Profile - Create*** and ***User Profile - Update*** incident types.
```
Comment (roysagi) [inline Packs/Okta/ReleaseNotes/2_2_0.md:8]:
```suggestion
- Updated the mapper to use the **User Profile - Create** and **User Profile - Update** incident types.
```

File: Packs/Salesforce/ReleaseNotes/1_1_0.md [hunk 14]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Salesforce IAM
+- Added the **Allow enabling users** parameter.
```
Comment (roysagi) [inline Packs/Salesforce/ReleaseNotes/1_1_0.md:4]:
```suggestion
- Added the *Allow enabling users* integration parameter.
```

File: Packs/Salesforce/ReleaseNotes/1_1_0.md [hunk 15]
```diff
@@ -0,0 +1,5 @@
...
+- Added support for customizing fields mapping differently in ***iam-create-user*** and ***iam-update-user*** commands.
```
Comment (roysagi) [inline Packs/Salesforce/ReleaseNotes/1_1_0.md:5]:
```suggestion
- Added support for customizing fields mapping differently in the ***iam-create-user*** and the ***iam-update-user*** commands.
```

File: Packs/ServiceNow/ReleaseNotes/2_2_0.md [hunk 16]
```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+##### ServiceNow IAM
+- Added the **Allow enabling users** parameter.
```
Comment (roysagi) [inline Packs/ServiceNow/ReleaseNotes/2_2_0.md:4]:
```suggestion
- Added the *Allow enabling users* integration parameter.
```

File: Packs/ServiceNow/ReleaseNotes/2_2_0.md [hunk 17]
```diff
@@ -0,0 +1,9 @@
...
+- Added support for customizing fields mapping differently in ***iam-create-user*** and ***iam-update-user*** commands.
```
Comment (roysagi) [inline Packs/ServiceNow/ReleaseNotes/2_2_0.md:5]:
```suggestion
- Added support for customizing fields mapping differently in the ***iam-create-user*** and the ***iam-update-user*** commands.
```

File: Packs/ServiceNow/ReleaseNotes/2_2_0.md [hunk 18]
```diff
@@ -0,0 +1,9 @@
...
+
+#### Mappers
+##### User Profile - ServiceNow (Outgoing)
+- Updated the mapper to use the ***User Profile - Create*** and ***User Profile - Update*** incident types.
```
Comment (roysagi) [inline Packs/ServiceNow/ReleaseNotes/2_2_0.md:9]:
```suggestion
- Updated the mapper to use the **User Profile - Create** and **User Profile - Update** incident types. 
```

File: Packs/Slack/ReleaseNotes/1_4_0.md [hunk 19]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Slack IAM
+- Added the **Allow enabling users** parameter.
```
Comment (roysagi) [inline Packs/Slack/ReleaseNotes/1_4_0.md:4]:
```suggestion
- Added the *Allow enabling users* integration parameter.
```

File: Packs/Slack/ReleaseNotes/1_4_0.md [hunk 20]
```diff
@@ -0,0 +1,5 @@
...
+- Added support for customizing fields mapping differently in ***iam-create-user*** and ***iam-update-user*** commands.
```
Comment (roysagi) [inline Packs/Slack/ReleaseNotes/1_4_0.md:5]:
```suggestion
- Added support for customizing fields mapping differently in the ***iam-create-user*** and the ***iam-update-user*** commands.
```

**Pull request merged**